static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-epic-liveblog.js

```diff
@@ -1,11 +1,6 @@
 // @flow
-const lastSentenceTemplate = (highlightedText?: string, supportURL: string, ctaText?: string) =>
-    `${
-        highlightedText
-            ? `<span className="contributions__highlight">${highlightedText}</span>`
-            : ''
-    }
-    <div class="component-button--liveblog-container">
+const lastSentenceTemplate = (supportURL: string, ctaText?: string) =>
```

Comment:
I guess we could rename this to something like `ctaTemplate`?

Reply:
good point!